File: apache-tomcat-10/src/main/java/io/github/robertomike/baradum/configs/AutoConfigurationSpring3.java
package io.github.robertomike.baradum.configs;

import io.github.robertomike.baradum.Baradum;
import io.github.robertomike.baradum.requests.ApacheTomcatRequest;
import jakarta.servlet.http.HttpServletRequest;

public class AutoConfigurationSpring3 {
    public AutoConfigurationSpring3(HttpServletRequest request) {
        Baradum.setRequest(new ApacheTomcatRequest(request));
    }
}

File: apache-tomcat-10/src/main/java/io/github/robertomike/baradum/configs/AutoConfigurationSpring3.kt
package io.github.robertomike.baradum.configs

import io.github.robertomike.baradum.Baradum.Companion.setRequest
import io.github.robertomike.baradum.requests.ApacheTomcatRequest
import jakarta.servlet.http.HttpServletRequest

class AutoConfigurationSpring3(request: HttpServletRequest) {
    init {
        setRequest(ApacheTomcatRequest(request))
    }
}
